### ISSUE TITLE
Chunk loading issues on bedrock

### ISSUE THREAD

**Issue by hw2007** [GeyserMC/Geyser#3845]
### Describe the bug

Sometimes on bedrock clients, chunks will stop loading. It seems to happen when a bedrock player is sitting in the back (passenger) part of a boat, and another player is driving. The player then has to relog to fix it.

### To Reproduce

1. Put one player into a boat
2. Put a bedrock player into the back of the boat
3. Make the first player drive the boat into unloaded chunks
4. Notice how on the bedrock passenger, the chunks will sometimes (not all the time) fail to load.

### Expected behaviour

Chunks should never stop loading.

### Server Version and Plugins

Minecraft 1.20, Fabric loader 0.14.21, Fabric installer version 0.11.2

### Geyser Dump

https://dump.geysermc.org/owMb9b4T2XvyMoDuE4o6bXATuL0RFAOm

### Geyser Version

Geyser version 2.1.1-SNAPSHOT (git-master-4cd4729)

### Minecraft: Bedrock Edition Device/Version

Observed on iOS, but untested on other devices

**Comment by Kas-tle** [comment]
Could you please test this on another device with a higher render distance? I think this may just be a duplicate of https://github.com/GeyserMC/Geyser/issues/3809.

**Comment by hw2007** [comment]
> Could you please test this on another device with a higher render distance? I think this may just be a duplicate of #3809.

The render distance was 18 chunks, but I can test on a console tomorrow

**Comment by Kas-tle** [comment]
What is the server render distance?

**Comment by hw2007** [comment]
> What is the server render distance?

Server distance is 24 chunks

**Comment by Kas-tle** [comment]
Then this is likely a duplicate of the mentioned issue if it does not occur when your client render distance meets or exceeds 24 chunks.

**Comment by hw2007** [comment]
> Then this is likely a duplicate of the mentioned issue if it does not occur when your client render distance meets or exceeds 24 chunks.

Okay, I will test on a device set to 28 tommorow, and report back.

**Comment by hw2007** [comment]
> > What is the server render distance?
> 
> Server distance is 24 chunks

Yup, this is a duplicate, it doesn't happen with higher distance

**Comment by Kas-tle** [comment]
I will close then. Please follow the other issue for updates.
